Remotion/Core/UnitTests/Core.UnitTests.csproj

```diff
@@ -57,7 +57,7 @@
     <CheckForOverflowUnderflow>false</CheckForOverflowUnderflow>
     <ConfigurationOverrideFile>
     </ConfigurationOverrideFile>
-    <DefineConstants>DEBUG;TRACE</DefineConstants>
+    <DefineConstants>DEBUG;TRACE;NETFRAMEWORK;</DefineConstants>
```

Comment:
I chose `NETFRAMEWORK` here, just like in the last RMTP tasks because thats the predefined preprocessor symbol for full framework when using SDK style projects. Imo this should stay and the occurrences in RM-7800 should be changed. (Refer to https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives#conditional-compilation )

Reply:
Ah, I C.

That's fine, it was just a hipshot for me. Could you please sync with @a-ctor  and @raffaelfoidl to use `NETFRAMEWORK` consistently despite what's written in the Jira tickets?